docs/ytt-data-values.md

```diff
@@ -1,27 +1,32 @@
 ## ytt @data/values
 
-- [Defining data values](#defining-data-values)
+The standard way to externalize configuration values is to declare them as "Data Values"
+and then reference those values in templates.
+
+- [Defining and Using Data Values](#declaring-and-using-data-values)
```

Comment:
`[Declaring and Using Data Values](#declaring-and-using-data-values)`

Reply:
whoops! Thank you!